harvdev_utils/char_conversions/greek_to_sgml.py

```diff
@@ -1,24 +1,30 @@
-"""
-.. module:: greek_to_sgml
-   :synopsis: A module to convert Greek characters to FlyBase SGML for writing proformae.
+"""Module:: greek_to_sgml.
+
+Synopsis:
+    A module to convert Greek characters to FlyBase SGML for writing proformae.
+
+Author(s):
+    Gil dos Santos dossantos@morgan.harvard.edu, Chris Tabone ctabone@morgan.harvard.edu
 
-.. moduleauthor:: Gil dos Santos dossantos@morgan.harvard.edu
 """
 
 import re
 
 
 def greek_to_sgml(input_string):
-    """A function to convert Greek characters into FlyBase SGML for writing proformae.
-    e.g. \\u03B1 -> &agr;
+    r"""A function to convert Greek characters into FlyBase SGML for writing proformae.
```

Comment:
I want to flag this "weirdness". The "r" at the beginning of the docstring is required to avoid this warning:
`D301 Use r""" if any backslashes in a docstring`

Reply:
I confirmed that this function still works as expected (one quick test).